src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionFetchTestResultViewModel.swift

```diff
@@ -0,0 +1,95 @@
+//
+// 🦠 Corona-Warn-App
+//
+//  ExposureSubmissionFetchTestResultViewModel.swift
+
+import Foundation
+import UIKit
+
+class ExposureSubmissionFetchTestResultViewModel {
+
+	// MARK: - Init
+	
+	init(
+		onQRCodeButtonTap: @escaping () -> Void,
+		onTANButtonTap: @escaping () -> Void,
+		onHotlineButtonTap: @escaping () -> Void
+	) {
+		self.onQRCodeButtonTap = onQRCodeButtonTap
+		self.onTANButtonTap = onTANButtonTap
+		self.onHotlineButtonTap = onHotlineButtonTap
+	}
+	
+	// MARK: - Internal
+	
+	let onQRCodeButtonTap: () -> Void
+	let onTANButtonTap: () -> Void
+	let onHotlineButtonTap: () -> Void
+
+	var dynamicTableData: DynamicTableViewModel {
```

Comment:
Should be called `dynamicTableViewModel` because this is of type `DynamicTableViewModel` and not `Data`

Reply:
good point. will fix this.

Reply:
fixed